core/src/test/java/org/apache/accumulo/core/classloader/URLClassLoaderFactory.java

```diff
@@ -54,19 +54,4 @@ public ClassLoader getClassLoader(String contextName) {
       throw new IllegalArgumentException("Error creating URL from contextName: " + contextName, e);
     }
   }
-
```

Comment:
There were other changes in URLClassLoaderFactory as part of 3678 that could be backed out, might be easier to revert.

Reply:
I reverted this class back to what it was before 3678.